platform/linux-generic/odp_timer.c

```diff
@@ -298,31 +308,59 @@ static odp_timer_pool_t timer_pool_new(const char *name,
 		return ODP_TIMER_POOL_INVALID;
 	}
 
+	/* Find timer pool that has not been used for a while, or is used least recently.
+	 * This ensures that inline scan of an old timer pool has completed and it's memory
```

Comment:
Typo: `it's`. Copyright year can also be updated.

Reply:
Fixed in v2